CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ include(ALPSCompilerVersionCheck)
 include(ALPSCompilerTweaks)
 
 include(ALPSEnableMPI)
+include(ALPSEnableEigen)
```

Comment:
Isn't the CMake convention not usually: `Find<Packagename>.cmake`? Why the deviation here?

Reply:
because it's bundled I suppose. Find<PackageName>.cmake is for external dependency.